integration/settings.gradle

```diff
@@ -0,0 +1,2 @@
+rootProject.name = 'java-interop'
```

Comment:
Should we move this all into a Java-specific sub-directory? It seems odd to call this directory all "java-interop" since we expect it to house all languages.

Reply:
I don't think it's necessary since setting.gradle is for Java specific. Also since proto files are under gradle directory tree, moving all gradle files to a sub-directory makes the path to proto files longer. E.g for now the path is "src/main/proto", it becomes "java_project/src/main/proto" after moving.